### PR TITLE
git: Add commit body's sentences restrictions

### PR DESCRIPTION
**Pull request by carpawell**
I need legal grounds to complain about it.

### CODE REVIEW

File: git.md [hunk 1]
```diff
@@ -62,7 +62,9 @@ The following format is expected:
   Long description with references, links, test data and whatever else is
   needed to explain what, why and how is changed (use up to 80-character
   columns for word-wrapping, any attachment like an application log,
-  external link, utility output is an exception).
+  external link, utility output is an exception). It should consist of
+  sentences (or a single sentence) that start with a capital letter and end
```
Comment (AnnaShaleva) [inline git.md:66]: s/`that start`/`that starts`?
s/`end`/`ends`?

Basically, it should just obey at least some basic punctuation/grammar rules.

Comment (carpawell) [reply git.md:66]: in "sentences that start" i would say that the subject is "sentences" not "that". but i am not sure

> Basically, it should just obey at least some basic punctuation/grammar rules.

dont mind paraphrasing. can you suggest smth? should a header "obey at least some basic punctuation/grammar rules" too?

Comment (AnnaShaleva) [reply git.md:66]: >  can you suggest smth?

Maybe something like " It should consist of sentences (or a single sentence) that follows at least some basic punctuation/grammar rules, e.g. start with a capital letter and end with a period."

> should a header "obey...

It is desirable, but we don't use dots in the end of commit headers (likely, for the sake of space saving).

Comment (carpawell) [reply git.md]: > > should a header "obey...
>
> It is desirable, but we don't use dots in the end of commit headers (likely, for the sake of space saving).

Yes, that is why i am not sure "some basic punctuation/grammar rules" is understandable enough. Change it to be your version

**Review by AnnaShaleva — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**